test/end-to-end/auth.js

```diff
@@ -116,7 +116,7 @@ describe("Login endpoint test", () => {
                     .send();
 
                 expect(res.status).toBe(HTTPStatus.UNAUTHORIZED);
-            }
+            },
```

Comment:
This seems like it's adding a comma after the last argument of functions, remove that rule